guardrails/cli/hub/install.py

```diff
@@ -237,11 +237,27 @@ def do_nothing_context(*args, **kwargs):
     with loader(dl_deps_msg, spinner="bouncingBar"):
         install_hub_module(module_manifest, site_packages, quiet=quiet)
 
+    if module_manifest.tags.has_guardrails_endpoint:
+        install_local_models = typer.confirm(
```

Comment:
We should test this with a `| y` or something to make sure confirmation can be automated in CI scripts.

Reply:
Or add an explicit flag to override this

Reply:
added tests here, hopefully this covers what you mean. The typer.confirm provides a Y/N option